python/svix/api.py

```diff
@@ -125,15 +125,15 @@ def _api(self) -> t.Generator[ApiClass, None, None]:
 class Authentication(ApiBase[AuthenticationApi]):
     _ApiClass = AuthenticationApi
 
-    def dashboard_access(self, app_id: str) -> DashboardAccessOut:
+    def dashboard_access(self, app_id: str, idempotency_key: str = "") -> DashboardAccessOut:
         with self._api() as api:
             return api.get_dashboard_access_api_v1_auth_dashboard_access_app_id_post(
-                app_id=app_id, _check_return_type=False
+                app_id=app_id, _check_return_type=False, idempotency_key=idempotency_key
             )
 
-    def logout(self) -> None:
+    def logout(self, idempotency_key: str = "") -> None:
```

Comment:
```suggestion
    def logout(self, *, idempotency_key: str = "") -> None:
```
For all of them.

What it essentially does: forces people to only set it by key, and never as an argument.

So:
```python
# This works (and shouldn't):
obj.logout("some_idempotency_key")
# With my change it'll have to be called like this:
obj.logout(idempotency_key="some_idempotency_key")
```